lib/pharos/phases/configure_service_account.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module Pharos
+  module Phases
+    class ConfigureServiceAccount < Pharos::Phase
+      title "Configure 'pharos-admin' service account"
+
+      ADMIN_USER = 'pharos-admin'
+      KUBECONFIG_PARAM = '--kubeconfig=/etc/kubernetes/admin.conf'
+
+      def call
+        create_service_account
+        create_cluster_role_binding
+
+        config = build_config
+
+        if config_file.exist?
+          existing_config = Pharos::Kube::Config.new(config_file.read)
+          config << existing_config
+        end
+
+        config_file.write(config.dump, overwrite: true)
+        config_file.chmod('0600')
+
+        validate
+      end
+
+      def validate
+        transport.exec!('kubectl get -n kube-system serviceaccount/pharos-admin')
+      end
+
+      def config_file
+        @config_file ||= transport.file(File.join(home_kube_dir.path, 'config'))
+      end
+
+      def home_kube_dir
+        transport.file(transport.file('~/.kube').readlink(escape: false, canonicalize: true)).tap do |dir|
+          transport.exec!("mkdir '#{dir}' && chmod 0700 '#{dir}") unless dir.exist?
+        end
+      end
+
+      def create_service_account
+        transport.exec!("sudo kubectl get #{KUBECONFIG_PARAM} -n kube-system serviceaccount/#{ADMIN_USER} || sudo kubectl #{KUBECONFIG_PARAM} -n kube-system create serviceaccount #{ADMIN_USER}")
```

Comment:
IMO we don't need sudo for `kubectl`.

Reply:
Ok, maybe we need because this points to root readable kubeconfig?

Reply:
`error: Error loading config file "/etc/kubernetes/admin.conf": open /etc/kubernetes/admin.conf: permission denied`